data/core/macros/names.cfg

```diff
@@ -173,6 +174,7 @@ suffix_male=af|al|lih|bair|sur|bi|lah|at|sim|ma|ih|yl|iq|raj|mam|man|ya|zid|riya
 #enddef
 
 #define LIZARD_NAMES
+    # Saurian name definition: no Context-free_grammar: male_name_generator/female_name_generator missing
```

Comment:
There _is_  context-free grammar for saurians - see the name_generator key. I'm not sure whether it's actually used though, could be a bug there.

Reply:
yeah that part of the patch gets rejected when trying to apply it against trunk; maybe it was added sometime after 1.14?